src/capture.js

```diff
@@ -439,15 +439,25 @@ Capture.ios8AndGreaterScrollFix = function(doc, callback) {
     // Using `getElementsByTagName` here because grabbing head using
     // `document.head` will throw exceptions in some older browsers (iOS 4.3).
     var head = doc.getElementsByTagName('head');
-    // Be extra safe and guard against `head` not existing.
+    var docViewport = doc.querySelector('meta[name="viewport"]');
```

Comment:
@donnielrt so this doc is going to be the main `document`, and not the captured doc. We probably want to query the capturing document here - this won't be able to see if there is a viewport tag all of the markup that is currently shoved in the plaintext tag.